File: usedesk_sdk/src/main/java/ru/usedesk/sdk/internal/data/framework/api/standard/entity/response/ErrorResponse.java
package ru.usedesk.sdk.internal.data.framework.api.standard.entity.response;

public class ErrorResponse extends BaseResponse {

    public static final String TYPE = "@@redbone/ERROR";

    private String message;
    private int code;

    public ErrorResponse() {
        super(TYPE);
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }
}

File: chat-sdk/src/main/java/ru/usedesk/chat_sdk/data/repository/api/loader/socket/_entity/error/ErrorResponse.kt
package ru.usedesk.chat_sdk.data.repository.api.loader.socket._entity.error

import ru.usedesk.chat_sdk.data.repository.api.loader.socket._entity._extra.BaseResponse

internal class ErrorResponse : BaseResponse() {
    var message: String? = null
    var code: Int? = null

    companion object {
        const val TYPE = "@@redbone/ERROR"
    }
}